packages/ckeditor5-ui/src/editorui/editorui.ts

```diff
@@ -649,3 +651,30 @@ function getToolbarDefinitionWeight( toolbarDef: FocusableToolbarDefinition ): n
 
 	return weight;
 }
+
+export function initMenuBar( editorUI: EditorUI & { view: EditorUIView & { menuBarView: MenuBarView } } ): void {
```

Comment:
Please add docs.

Reply:
@scofalik I was wondering if this is a good place for that function. I'd move it to `menubar/utils.js`. WDYT?

Reply:
At first glance, it feels to me that `menubar/utils.ts` is a better place as the function is about menu bar and editor UI is just a dependency here. Additionally, `editorui.ts` does not export any utils at this moment.

Reply:
But TBH, I don't see a reason why inline editor should not be able to have menu bar, and at that point we will probably move this into `EditorUI` class.

Reply:
I'd consider making it public-internal to avoid breaking changes. I don't believe there is a case where this util could be useful for integrators at this moment :thinking:.